ocaml/xapi/firewall.ml

```diff
@@ -87,11 +87,12 @@ let service_type_to_service_info = function
       }
 
 module type FIREWALL = sig
-  val update_firewall_status : service:service_type -> status:status -> unit
+  val update_firewall_status :
+    ?interfaces:string list -> service_type -> status -> unit
```

Comment:
Please document what type of string should the list in `interfaces` contain, it's not obvious